src/client/pydaos/__init__.py

```diff
@@ -25,42 +25,69 @@
 import sys
 import atexit
 
+dc = None
```

Comment:
should this be _dc to make it invisible to the package user?

Reply:
Cont objects in pydaos_core take a reference to this to avoid it being deleted so it's deliberately exported.